source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -45,9 +45,17 @@ def delete_journeys(responses, request):
     if request.get('debug', False):
         return
 
+    keep_jo = request.get("_jo", False)
+
+    def to_delete(journey):
+        if keep_jo:
+            return to_be_deleted(journey) and "olympics" not in journey.tags
```

Comment:
Same, I'd avoid being too specific:
```suggestion
            return to_be_deleted(journey) and "protected" not in journey.tags
```